data.json

```
@@ -1300,5 +1300,11 @@
     "githubUsername": "Mo-Shakib",
     "imageUrl": "https://user-images.githubusercontent.com/50780268/131604251-f07f6811-d795-448f-a7b8-0f13b58c602f.png",
     "issueNumber": 675
+  },
+  { 
+    "name": "Komal Pal",
+    "githubUsername": "Komal7209",
+    "imageUrl":"https://user-images.githubusercontent.com/33154959/131854111-cff5bad4-fd84-46e8-abf0-3ce469e83fca.png",
```

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/33154959/131854111-cff5bad4-fd84-46e8-abf0-3ce469e83fca.png",
```
If you want leave the image as it is please update the line as shown above

---

I have now made changes as per requested